game.rb

```diff
@@ -0,0 +1,39 @@
+require 'json'
+require_relative './item'
+require_relative './label'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(*args, last_played_at, multiplayer)
+    super(*args)
+    @last_played_at = last_played_at
+    @multiplayer = multiplayer
+  end
+
+  def to_json(*args)
+    {
+      JSON.create_id => self.class.name,
+      'id' => @id,
+      'title' => @title,
+      'publish_date' => @publish_date,
+      'archived' => @archived,
+      'multiplayer' => @multiplayer,
+      'last_played_at' => @last_played_at,
+      'author' => @author ? ['id' => @author.id, 'first_name' => @author.first_name, 'last_name' => author.last_name] : nil,
+      'label' => @label ? ['id' => @label.id, 'title' => @label.title, 'color' => @label.color] : nil,
+      'genre' => @genre ? ['id' => @genre.id, 'name' => @genre.name] : nil,
+      'source' => @source ? ['id' => @source.id, 'name' => @source.name] : nil
+    }.to_json(*args)
+  end
+
+  def self.json_create(game)
+    new(game['id'], game['title'], game['publish_date'], game['archived'],
+        game['multiplayer'], game['last_played_at'],
+        game['author'], game['label'], game['genre'], game['source'])
+  end
+
+  def can_be_archived?()
+    super && last_played_at > 1
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md#group-task:~:text=should%20return%20true%20if%20parent%27s%20method%20returns%20true%20AND%20if%20last_played_at%20is%20older%20than%202%20years), this method suppose to return true if parent's method returns true AND if last_played_at is older than 2 years. So please fix that.